src/SiteLink.php

```diff
@@ -18,10 +18,18 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Michał Łazowik
+ * @author Thiemo Mättig
```

Comment:
I fail to see an "honest" question. Why are you asking if the added `@author` is unrelated? You can see it is unrelated. It takes you seconds to look at the history and blame of this class and see that I'm a co-author of relevant parts of the code.

Why are you adding that suggestive sentence about me not knowing how `@author` tags should be used?